telluric/vectors.py

```diff
@@ -219,8 +219,20 @@ def delegated_(self_):
 
         elif item in GEOM_BINARY_PREDICATES:
             def delegated_predicate(self_, other):
-                return getattr(self_.get_shape(self_.crs), item)(
-                    other.get_shape(self_.crs))
+                relationship = getattr(
+                    self_.get_shape(self_.crs), item
+                )(other.get_shape(self_.crs))
+                if item not in ['intersects']:
```

Comment:
This can only happen with `intersects`? What about other predicates?

Reply:
I've discovered this issue when trying to "rasterize" part of the raster with (-180, -90, 180, 90) bounds and I'm pretty sure that it can happens for other predicates too. Do you think we shouldn't merge it in a current state?

Reply:
I think we should follow the "transform 1 to 2, and if not, try 2 to 1" approach more generally - or perhaps check for nan values. I understand the urge so I'm not going to block the pull request, just a thought.

Reply:
let me jump in here, transform 1 to 2 and 2 to 1 are not always commutative, for example `v.within(v2)` might be true while `v2.within(v)` might be False